components/PageSelector.tsx

```diff
@@ -17,7 +17,7 @@ export default function PageSelector(
           max={props.lastPage}
           value={props.currentPage}
           // @ts-ignore: this is valid HTML
-          onchange="this.form.submit()"
+          onchange={(e) => this.form.submit()}
```

Comment:
```suggestion
          onchange={() => this.form.submit()}
```
If the event parameter isn't used, it can be omitted.